packages/cli/src/utils/registry/index.ts

```diff
@@ -155,7 +155,7 @@ async function fetchRegistry(paths: string[]) {
 
 		return results;
 	} catch (error) {
-		console.log(error);
+		console.error(error);
```

Comment:
I'm not sure if this change is okay, since it's followed by another `throw new Error` statement. 
Should we remove this? keep the original `console.log`?